### DISCUSSION TITLE
Incomplete SVG example (uibuilder dynamic SVG example - no framework needed) ?

### DISCUSSION THREAD

**Discussion by Kilill** [TotallyInformation/node-red-contrib-uibuilder#229] [Q&A] [answered]
Seems that the svg example "uibuilder dynamic SVG example - no framework needed" at https://flows.nodered.org/flow/89487d20c635d54d9b98d963656ce429 is incomplete ?

**Comment by TotallyInformation** [comment]
Sorry. I replied in the issue you raised on the other repo. There is a built-in template for Svelte with a simple flow in the "Templates" example in the Node-RED library.

Can't quite remember why I thought an external template for Svelte might be a good idea. Perhaps I was thinking of expanding on what was in the internal template. Or maybe I was preparing for the work on the UIBUILDER backlog for expanding the capabilities of the templates.

**Reply by Kilill** [reply]
No problem what so ever :-) , got both your updates....

**Reply by Kilill** [reply]
eh... this issue was not about the svelte example but about but about the SVG example seems the jason code is cut of at the end

**Reply by TotallyInformation** [reply]
Looks like there might be something wrong with the flows site.

**Reply by TotallyInformation** [reply, marked as answer]
Try the original GIST here: https://gist.github.com/TotallyInformation/89487d20c635d54d9b98d963656ce429
